payloads/github_api/repos_dlang_dmd_pulls?state=open.json

```diff
@@ -0,0 +1,9128 @@
+[
```

Comment:
Maybe it's possible to gzip the files? Vibe.d serves pre-compressed files by default.